Detections/AzureDevOps/AzDOHistoricPrPolicyBypassing.yaml

```diff
@@ -0,0 +1,44 @@
+id: 4d8de9e6-263e-4845-8618-cd23a4f58b70
+name: Azure DevOps Pull Request Policy Bypassing - Historic Allowlist
+description: |
+  'This detection builds a Allowlist of historic PR policy bypasses and compares to recent history, flagging a non manually allowlisted, non historic pull request bypass.'
+severity: Medium
+queryFrequency: 2h
+queryPeriod: 14d
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+  - Execution
+  - Impact
```

Comment:
Please use the MITRE ATT&CK full names with no spaces. Here Escalation would be PrivilegeEscalation, there is no Impact so will need to remove or change to another and we also require Techniques in Detections - https://attack.mitre.org/techniques/enterprise/

Format as follows -

relevantTechniques:

T1078
T1110

Reply:
fixed in next commit